Modules/_ctypes/cfield.c

```diff
@@ -658,14 +516,17 @@ vBOOL_set(void *ptr, PyObject *value, Py_ssize_t size)
 }
 
 static PyObject *
-vBOOL_get(void *ptr, Py_ssize_t size)
+v_get(void *ptr, Py_ssize_t size)
 {
+    assert(NUM_BITS(size) || (size == sizeof(short int)));
     return PyBool_FromLong((long)*(short int *)ptr);
 }
 
+/* bool ('?'): _Bool */
 static PyObject *
 bool_set(void *ptr, PyObject *value, Py_ssize_t size)
 {
+    assert(NUM_BITS(size) || (size == sizeof(bool)));
```

Comment:
Should this be `_Bool` too?

Reply:
Now that I use `bool` for internal logic (`_ctypes_fixint_fielddesc(..., bool is_signed)`), I'll include `<stdbool.h>` and use `bool` throughout. I predict that future versions of C will push toward `bool` as the preferred spelling, too.